ui/Modal/Funding/LinkAddress.svelte

```diff
@@ -1,9 +1,9 @@
 <script lang="typescript">
-  import EthToRadicle from "./Link/EthToRadicle.svelte";
-  import SavedToRadicle from "./Link/SavedToRadicle.svelte";
-  import RadicleToEth from "./Link/RadicleToEth.svelte";
   import { Remote } from "../../DesignSystem/Component";
 
+  import { Avatar, Button, Emoji, Icon } from "../../DesignSystem/Primitive";
+  import { Copyable, TxButton } from "../../DesignSystem/Component";
```

Comment:
It’s now possible (and preferable) to use absolute imports from the Design System.

```suggestion
  import { Avatar, Button, Emoji, Icon } from "ui/DesignSystem/Primitive";
  import { Copyable, TxButton } from "ui/DesignSystem/Component";
```

Reply:
Fixed